addons/website_sale/static/src/js/website_sale.js

```diff
@@ -1,10 +1,10 @@
 odoo.define('website_sale.cart', function (require) {
 'use strict';
 
+const { sessionStorage } = require("@web/core/browser/browser")
```

Comment:
Why do you add this import?

Reply:
@chevalierv 
it is in the opw mentioned in the fist commit.
when a user has set his chrome to not store anything via the "Manage on-device site data", accessing window.[storage] crashes.
Our browser.js interface catches the error and makes a ram storage instead

Reply:
I saw your commit message. 

Sorry for the confusion, I've checked the file in master and we don't use `sessionStorage` in it anymore. I didn't check that you were targeting `16.0`.